boa/core/test.py

```diff
@@ -352,6 +352,26 @@ def construct_metadata_for_test(recipedir_or_package, config):
     return m, hash_input
 
 
+def test_run_v2(test_yaml_path,):
+    PREFIX = "/Users/madhur/mambaforge/envs/boa/"
```

Comment:
PREFIX should be an input to the function

Reply:
Done